docs/docs/getting-started/quickstart.md

```
@@ -1,204 +1,114 @@
----
+Altimate Code searches for `profiles.yml` in this order: `DBT_PROFILES_DIR` env var → project root (next to `dbt_project.yml`) → `<home>/.dbt/profiles.yml`. It reads your dbt profiles and creates warehouse connections automatically. You'll see output like:
 
-## Step 3: Connect Your Warehouse
-
-### Auto-discover with `/discover`
-
-> Skip this step if you want to work locally. You can always run `/discover` later.
-
-```bash
-altimate /discover
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)